geomstats/learning/em_expectation_maximization.py

```diff
@@ -0,0 +1,265 @@
+"""Expectation maximisation algorithm."""
+
+import logging
+
+import geomstats.backend as gs
+from geomstats.geometry.poincare_ball \
+    import GaussianDistribution, Normalization_Factor_Storage
+from geomstats.learning.frechet_mean import FrechetMean
+
+
+EM_CONV_RATE = 1e-4
+MINIMUM_EPOCHS = 10
+DEFAULT_MAX_ITER = 100
+DEFAULT_LR = 5e-2
+DEFAULT_CONV_FACTOR = 1e-4
+DEFAULT_TOL = 1e-2
+ZETA_LOWER_BOUND = 5e-2
+ZETA_UPPER_BOUND = 2.
+ZETA_STEP = 0.001
+PDF_TOL = 1e-15
+SUM_CHECK_PDF = 1e-4
+MEAN_MAX_ITER = 150
+
+
+class RiemannianEM():
```

Comment:
No ()

Reply:
Is there a base scikit-learn class from which this class could inherit and follow the API?